style.css

```diff
@@ -92,7 +98,92 @@
   display: flex;
 }
 
-@media only screen and (max-width: 650px) {
+.btn {
+  color: #6070ff;
+  text-decoration: none;
+  border-radius: 7px;
+  padding: 0.6rem;
+  border: 1px solid #6070ff;
+  font-size: 16px;
+  font-weight: bold;
+}
+
+.btn:hover {
+  color: #fff;
+  background-color: #6070ff;
+  transition: 1s;
+}
+
+.mt-4 {
+  height: 4rem;
+}
+
+.mt-3 {
+  height: 3rem;
+}
+
+.content h5 {
+  text-transform: uppercase;
+  color: #6070ff;
+  font-size: 18px;
+  margin-top: 2.5rem;
+}
+
+.experience {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  width: 60%;
+  justify-items: center;
+  gap: 5px;
+}
+
```

Comment:
- [ ] Make sure on using the same font family, margins color, and spaces as specified in the original design.
- [ ] [OPTIONAL] The live demo link is not correctly working. I think it will be good to fix it and add it to the readme file.